### PR TITLE
Apollo create transactions

### LINKED ISSUE #823: 🔧 [Refactor] Implement Login Call createTransactions on Apollo
## 🔧 Refactor ticket

Implement Login Call createTransactions on Apollo

### PR DESCRIPTION
**Pull request by einhornimmond**
## 🍰 Pullrequest
- based on #893 so better merge this first
- implement create send coins transaction on apollo 
- [x] get public keys for sender and recipiant (currently only for local transfers)
- [x] validate input 
- [x] find and install protobuf module 
- [x] add sign function on login server
- [x] sign transaction 
- [x] write transaction block into db
- [x] update balance of user
- [x] test code  

### Issues
- fixes #823

### CODE REVIEW

**Comment by Mogge** [comment]
@einhornimmond  Please clean up this PR. There are all the commits we did in the last weeks included. This makes it impossible to review the PR (112 files changed)

**Comment by einhornimmond** [comment]
Yes I have noticed that I must make some kind of error by rebasing/merge with master.

**Comment by ulfgebhardt** [comment]
This could be helpful if needed: https://mattstauffer.com/blog/how-to-merge-only-specific-commits-from-a-pull-request/

File: .gitmodules [hunk 1]
```diff
@@ -34,3 +34,6 @@
 [submodule "login_server/dependencies/protobuf"]
 	path = login_server/dependencies/protobuf
 	url = https://github.com/protocolbuffers/protobuf.git
+[submodule "backend/src/proto"]
+	path = backend/src/proto
+	url = git@github.com:gradido/gradido_protocol.git
```
Comment (Elweyn) [inline .gitmodules:39]: This defines the database tables? Or the Blockchain?

**Review by Elweyn — APPROVED** [review]

**Pull request merged**